packages/muze-axis/src/cartesian-axis/interpolators/log.js

```diff
@@ -13,16 +13,21 @@ export default class LogInterpolator {
     }
 
     getScaleValue (domainVal) {
+        const scale = this._scale;
         if (domainVal <= 0) {
-            return 1;
+            return scale(scale.domain()[0]);
```

Comment:
What happens if scale.domain()[0] is a negative value?
Is it possible that it can happen?